### ISSUE TITLE
More descriptive ylabel for Series.hist()

### ISSUE THREAD

**Issue by clarkfitzg** [pydata/pandas#10485]
```
s = pd.Series(np.random.randn(100))
s.plot(kind='hist')
s.hist()
```

Running `s.plot(kind='hist')` labels the y axis with the word 'Degree'. A more informative label might be 'count', 'frequency', or 'density'.

Also, it seems strange that `s.plot(kind='hist')` and `s.hist()` produce plots with different labels / gridlines.

Looks like this came from PR #7809. cc @sinhrks, @jreback

**Comment by shoyer** [comment]
The name "Frequency" sounds more appropriate to me -- unless `normed=True`, in which case "Density" would be the right word.

**Comment by clarkfitzg** [comment]
+1 for 'Frequency'. That would make sense to me if `normed=True` or not. 'Density' makes me think of a probability density function.

**Comment by sinhrks** [comment]
Related to #9093. The reason I've added ``ylabel`` is for consistency with ``kind='kde'``. I'm willing to change it if there is better word choice (of course PR is welcome!).

**Comment by clarkfitzg** [comment]
Looks like `kind='kde'` labels it with 'Density'.

**Comment by sinhrks** [comment]
Closed by #10493. Thanks @clarkfitzg !
